src/mame/sony_news/cxd8452aq.cpp

```diff
@@ -0,0 +1,295 @@
+// license:BSD-3-Clause
+// copyright-holders:Brice Onken
+// thanks-to:Patrick Mackinlay
+
+/*
+ * Sony CXD8452AQ WSC-SONIC3 APbus interface and Ethernet controller controller
+ *
+ * TODO:
+ *  - Error-related interrupts
+ */
+
+#include "emu.h"
+#include "cxd8452aq.h"
+
+#define LOG_GENERAL (1U << 0)
```

Comment:
This is defined in `logmacro.h` and shouldn't be redefined here.

Reply:
Good catch. Fixed, thanks!